c++/src/kj/exception.c++

```diff
@@ -752,6 +756,10 @@ Exception::Exception(const Exception& other) noexcept
     file = ownFile.cStr();
   }
 
+  if (other.remoteTrace != nullptr) {
```

Comment:
Huh, I hadn't noticed this functionality of comparing a `String` to `nullptr` before.